jobs/pull-kubernetes-federation-e2e-gce-gci.sh

```diff
@@ -47,7 +47,14 @@ export KUBE_FASTBUILD=true
 ./hack/jenkins/build.sh
 
 # Push federation images to GCS.
-./build/push-federation-images.sh
+if [[ -e build/push-federation-images.sh ]]; then
```

Comment:
as I noted in the other PR, just remove these entirely. they're redundant. look at the build log [here](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/35765/pull-kubernetes-federation-e2e-gce/5/), see that it already calls `push-federation-images.sh`.